src/content_script.js

```diff
@@ -250,7 +238,28 @@ function addFacebookBadge (target, badgeClassUId, socialAction) {
 
 }
 
+// Add Event Listener actions/hooks to share badges
+function shareBadgeEventListenerInit(target, htmlBadgeDiv, options) {
+  if (!options.allowClickThrough) {
```

Comment:
Adding options object to this function to allow passive links to work. Active tracking links will remain in their freezed state.